docs/tutorials/create_your_first_library.md

```diff
@@ -0,0 +1,284 @@
+# Build a Pixano library
+
+## Context
+
+In this tutorial, we will build a library consisting of one dataset from a folder dataset stored in the `./assets/health_images/` folder with a unique subfolder `all/` which will later be considered as a split.
+
+It contains 10 images of human parts from several image sources (MRI, microscope, and high-resolution photos). A `metadata.jsonl` also provides annotations, bounding boxes and keypoints, associated to these images.
+
+## Build the Dataset
+
+### Describe your dataset
+
+To create a Pixano `Dataset`, a `DatasetBuilder` needs a pythonic description of the dataset item based on a Pydantic's [BaseModel](https://docs.pydantic.dev/latest/api/base_model/).
+
+To do so, Pixano provides the `pixano.datasets.DatasetItem` class that already has the attributes `id` and `split` to uniquely identify an item and categorize the items respectively.
+
+To define a custom `DatasetItem`, simply create a sublcass of `DatasetItem` that define all the features of your dataset.
+
+In our case we have:
+
+- `image_type`: a string metadata.
+- `image`: the unique view of the item that is an Image.
+- `objects`: the entites of an item. Think of it as a common identifier for multiple annotations.
+- `bbox`: the bounding boxes of an item.
+- `keypoints`: the keypoints of an item.
+
+This is how it can be defined in code:
+
+```python
+from pixano.datasets import DatasetItem
+from pixano.features import BBox, Entity, Image, KeyPoints
+
+
+class EntityWithCategory(Entity):
+    category: str
+
+
+class HealthDatasetItem(DatasetItem):
+    image: Image
+    objects: list[EntityWithCategory]
+    bbox: list[BBox]
+    keypoints: list[KeyPoints]
+    image_type: str
+```
+
+Notice that when multiple elements are attached to an item we use the `list` type.
+
+### Initialize a DatasetBuilder
+
+#### Use a built-in builder
+
+The Health dataset follows a **folder** structure that can be handled by Pixano:
+
+```
+root_folder/
+    split1/
+        view0.ext
+        view1.ext
+        ...
+        viewN.ext
+        metadata.jsonl
+    split2/
+        ...
+```
+
+Therefore the `pixano.datasets.builders.ImageFolderBuilder` can be used to construct the Pixano dataset as follows:
+
+```python
+from pathlib import Path
+from pixano.datasets import DatasetInfo
+from pixano.datasets.builders import ImageFolderBuilder
+
+
+builder = ImageFolderBuilder(
+    source_dir=Path("./assets/health_images"),
+    target_dir=Path("./pixano_library/health_images"),
```

Comment:
I suggest to use another name than health_images for target dir, maybe health_dataset, to avoid possible confusion

Reply:
Done